tests/workers/test_githuber.py

```diff
@@ -19,16 +19,21 @@ def test_execute(self, repo_name, repo_url):
         assert results is not None
         assert isinstance(results, dict)
         assert set(results.keys()) == {'details', 'status', 'summary'}
-        if not results['details'].keys():  # no github api token provided
-            return
```

Comment:
What if the token is not provided? Maybe [mark the test as skipped](https://docs.pytest.org/en/latest/skipping.html)?

Reply:
token is not provided by default in tests. It might fail because of rate limit, but we decided to give a shot to this approach since we should not exceed 60 requests per hour in tests.